vector/src/main/java/im/vector/app/core/utils/Emoji.kt

```diff
@@ -82,6 +85,9 @@ fun isSingleEmoji(string: String): Boolean {
  * @return true if the body contains only emojis
  */
 fun containsOnlyEmojis(str: String?): Boolean {
+    // Now rely on vanniktech library
+    return EmojiUtils.isOnlyEmojis(str)
+    /*
```

Comment:
Commenting old code is really necessary?

Reply:
I do the clean up